src/pages/hardware/developingAnIOX/messagingProtocol.tsx

```diff
@@ -0,0 +1,1361 @@
+                    <td>Data to transmit</td>
+                </tr>
+            </tbody>
+        </table>
+        <h3>RX Data (0x0C)</h3>
+        <p>Data sent from an IOX to the GO device. The GO will reply with an ACK. The contents of this payload may follow a higher level protocol structure such as <a href="/sdk/hardware/developing-an-iox/mime-protocol/">MIME</a>. The 0x0C message series start and end with a Information Type 1 - Packet Wrapper <a href="#iox-requeststatus-0x25">0x25 message</a>.</p>
+        <h4>Payload - RX Data</h4>
+        <table>
+            <thead>
+                <tr>
+                    <th>Byte #</th>
+                    <th>Byte Description</th>
+                </tr>
+            </thead>
+            <tbody>
+                <tr>
+                    <td>0-7</td>
+                    <td>Received data</td>
+                </tr>
+            </tbody>
+        </table>
+        <h3>Acknowledge (0x14)</h3>
+        <p>Sent by the GO to indicate that a message is being acknowledged. The ACK to an RX Data message (0x0C) could include 1 byte of data. This data is used for streaming flow control. When the 80% watermark of the receive buffer has been reached, the flow control bit will tell the IOX to hold off sending for 50 ms. The IOX will send the next frame at the end of this period and, depending on the flow control bit of the ACK, it will either continue sending or delay another 50 ms, thus repeating the process. The GO device will clear the flow control bit whenever the buffer is below the 20% watermark. If transferring data as part of a wrapped packet exchange, the streaming watermark can be ignored. The buffers will not overflow so long as the length limit and the modem result are honored. This byte is only sent when needed.</p>
+        <h4>Payload</h4>
+        <table>
+            <thead>
+                <tr>
+                    <th>Byte #</th>
+                    <th>Byte Description</th>
+                </tr>
+            </thead>
+            <tbody>
+                <tr>
+                    <td>0 - Bit 0</td>
+                    <td>
+                        0 = Clear to send more RX Data.
+                        <br></br>
+                        1 = Stop sending UART Data. Buffer 80% full, withhold next frame 50 ms. 
+                    </td>
+                </tr>
+                <tr>
+                    <td>1 - Bit 1-7</td>
+                    <td>Reserved</td>
+                </tr>
+            </tbody>
+        </table>
+        <h3>Application Specific Data (0x1C)</h3>
+        <p>Sent by the GO device after a packet wrapped passthrough message attempt to the server. A 'rejected' response from the modem typically means it is not connected. If the message is 'accepted', this means it was added to the modem's TCP socket buffer. It is NOT a confirmation that the message was successfully sent.</p>
+        <h4>Type 0: Modem Transmission Result</h4>
+        <table>
```

Comment:
we should have tables embedded in a div with the .table-container class until we come up with the final styling so it can be applied later on